.github/workflows/publish.yml

```diff
@@ -0,0 +1,36 @@
+name: zola build + publish
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@master
+    - name: get zola runtime
+      env:
+        ZOLA_VERSION: v0.7.0
```

Comment:
Looks like [0.8.0](https://github.com/getzola/zola/releases/tag/v0.8.0) has been released. Any reason not to upgrade?

Reply:
nope. Just gave it a try and seems to work just fine. thanks!